js-packages/@prestojs/ui-antd/src/widgets/FileWidget.tsx

```diff
@@ -31,12 +31,31 @@ export type UploadWidgetProps<FieldValue, T extends HTMLElement> = Omit<
     > & {
         input: UploadWidgetInputType;
         /**
-         * Number of files to allow. `null` means unlimited. Defaults to `1`.
+         * Number of files to allow. `null` means unlimited.
+         *
+         * If `limit` is `1` then after uploading a file the UI will change from an Add icon to an Edit icon and
+         * uploading a new file will replace the original.
+         *
+         * If `limit` is > `1` then once the limit has been reached the UI will disallow adding new files. To add a
+         * new file an existing one must be removed.
+         *
+         * If `limit` > `1` OR `multiple` is `true` then the value passed to `onChange` will be an array, otherwise
```

Comment:
limit > 1 and multiple==false is should really be an error, in which case that condition is just:

> If `multiple` is `true` then

Reply:
updated